configure.ac

```diff
@@ -86,6 +86,21 @@ AS_IF([test "x$enable_mpi" = "xyes"], [
 ])
 AM_CONDITIONAL([MPI], [test "x$enable_mpi" = "xyes"])
 
+# Enable SWIG
+AC_ARG_ENABLE(
+  [swig],
+  [AS_HELP_STRING([--enable-swig], [Enable SWIG])]
+)
+AS_IF([test "x$enable_swig" = "xyes"], [
+  AC_CHECK_PROG(JAVA, java, java)
```

Comment:
javac is a better check, since for the includes we need the full jdk rather than just the jre

Reply:
fixed